shepherd_bms/src/analyzer.cpp

```diff
@@ -177,6 +178,34 @@ void Analyzer::calcContDCL()
     }
 }
 
+void Analyzer::calcCCL()
+{
+    int16_t currentLimit = 0x7FFF;
+
+    for(uint8_t c = 0; c < NUM_CHIPS; c++)
+    {
+        for(uint8_t cell = 0; cell < NUM_CELLS_PER_CHIP; cell++)
+        {   
+            // Apply equation
+            uint16_t tmpCCL = (MAX_VOLT - ((bmsdata->chipData[c].openCellVoltage[cell] + VOLT_SAG_MARGIN) * 10000)) / (bmsdata->chipData[c].cellResistance[cell] * 10);
+            // Multiplying resistance by 10 to convert from mOhm to Ohm and then to Ohm * 10000 to account for the voltage units
```

Comment:
The comment and the actual equation disagree with each other. Also I do not think the 10000 multiplier is in the right spot. I believe MAX_VOLT and VOLT_SAG_MARGIN and 1x, openCellVoltage is 10000x, and I don't recall what cell resistance is. Somehow they just all need to be at the same multiplier

Reply:
i think looking at DCL equation, MAX_VOLT and VOLT_SAG_MARGIN are in the samw magnitude, and its just Open cell voltage that is * 10,000. So, i think this still needs some tweaking, looks like just changing MAX_VOLT multilper to be 10,000 not 1,000. If im missing something feel free to let me know